synapse/tools/modrole.py

```diff
@@ -115,5 +116,10 @@ async def main(argv, outp=s_output.stdout):
                     await cell.addRoleRule(roleiden, (False, perm), indx=0, gateiden=opts.gate)
     return 0
 
+async def _main(argv, outp=s_output.stdout):  # pragma: no cover
+    ret = await main(argv, outp=outp)
```

Comment:
`s_common.setlogging(logger, 'WARNING')` here?

Reply:
There are some tools which do set logging; and most tools do not. I wasn't trying to make those consistent, since they are kind of sprinkled around organically; and most CLI tool output is done with an `outp` object instead of a logger.